packages/design-system-tokens/src/tokens/System.Value.json

```diff
@@ -2663,7 +2663,7 @@
       "$extensions": {
         "com.figma": {
           "hiddenFromPublishing": false,
-          "scopes": ["GAP", "WIDTH_HEIGHT", "TEXT_CONTENT"],
+          "scopes": ["GAP", "TEXT_CONTENT", "WIDTH_HEIGHT"],
```

Comment:
Blerg. These changes will keep happening until v12 is released. See PR #3226 
I guess I can assign to a new milestone.

Reply:
I was wondering where those came from!